dataGenerator/src/main/java/com/smartbank/datagenerator/Config/KafkaTopicConfig.java

```diff
@@ -0,0 +1,38 @@
+package com.smartbank.datagenerator.Config;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.config.TopicBuilder;
+
+@Configuration
+public class KafkaTopicConfig {
+
+    @Value("${account.topic}")
+    private String accountTopic;
+
+    @Value("${account.topic}")
+    private String bank_workingTopic;
```

Comment:
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
Follow convention for variables

Reply:
done